<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigEditor</class>
 <widget class="QWidget" name="ConfigEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>508</width>
    <height>545</height>
   </rect>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="tabPosition">
      <enum>QTabWidget::TabPosition::North</enum>
     </property>
     <property name="tabShape">
      <enum>QTabWidget::TabShape::Rounded</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="elideMode">
      <enum>Qt::TextElideMode::ElideNone</enum>
     </property>
     <property name="usesScrollButtons">
      <bool>false</bool>
     </property>
     <property name="documentMode">
      <bool>true</bool>
     </property>
     <property name="tabsClosable">
      <bool>false</bool>
     </property>
     <property name="movable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="IO_tab">
      <attribute name="title">
       <string>IO</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Input LAS/LAZ Files</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QScrollArea" name="scrollArea">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>482</width>
            <height>237</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QTreeWidget" name="treeWidget"/>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Output</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QFormLayout" name="formLayout_2">
         <item row="1" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Scale:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <layout class="QFormLayout" name="formLayout_3">
           <item row="0" column="0">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>1:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QComboBox" name="scale_dropdown">
             <property name="editable">
              <bool>true</bool>
             </property>
             <property name="currentIndex">
              <number>3</number>
             </property>
             <item>
              <property name="text">
               <string>5000</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>7500</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>10000</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>15000</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>25000</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>50000</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>DPI</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QComboBox" name="dpi_dropdown">
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="editable">
            <bool>true</bool>
           </property>
           <property name="currentIndex">
            <number>2</number>
           </property>
           <item>
            <property name="text">
             <string>150</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>300</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>600</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>1200</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Output Directory:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLineEdit" name="out_dir_line_edit">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="out_dir_button">
             <property name="minimumSize">
              <size>
               <width>20</width>
               <height>20</height>
              </size>
             </property>
             <property name="text">
              <string>...</string>
             </property>
             <property name="icon">
              <iconset theme="QIcon::ThemeIcon::DocumentOpen"/>
             </property>
             <property name="autoRaise">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="line_2">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Processing Steps</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="1">
          <widget class="QCheckBox" name="process_tiles_checkbox">
           <property name="text">
            <string>Process tiles</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QCheckBox" name="extract_borders_checkbox">
           <property name="text">
            <string>Extract borders</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QCheckBox" name="combine_tiles_checkbox">
           <property name="text">
            <string>Combine Tiles</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Contours</string>
      </attribute>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
